### PR TITLE
new module: xkb_layouts

### PR DESCRIPTION
**Pull request by lasers**
This introduces a xkb switcher module using classes instead of setxkbmap [+xset] [+xkblayout-state].

Tested OK. Pls read the comments before proceeding with the review. Thx you.

Addresses #1816.

### CODE REVIEW

**Comment by ultrabug** [comment]
First of all, I'd rename it `xkeyboard_layout` which is basically what xkb stands for right?

I'll read the code and do a review.

**Comment by lasers** [comment]
It looks like `xkb` is `X keyboard extension` that covers lot of things. I like `xkb` or `xkb_layouts` more.

**Comment by lasers** [comment]
>@maximbaz Hey guys, I like that you found tools that make layout switching work better, but I'd like to question the need for deprecating setxkbmap — for example, I don't use variants, and in general I use this module purely to query the current layout (I have button_prev = button_next = None), and for my purposes setxkbmap seems to work quite well.

Try this readonly class... `setxkbmap` + `xset`. I don't like this, but it ought to do what you want... and it should deal with variants too... It might be enough to meet your needs. No scrolling though.

**Comment by maximbaz** [comment]
I think the point of #1816 was to get rid of `setxkbmap` + `xset`, if that approach is terrible and hacky and you have a chance to get rid of it in a new module, let's do it 🙂

**Comment by ultrabug** [comment]
Ok for `xkb_layouts` thanks

Since I've been slow and slacking, I've added two commits to default format and explicit missing binaries

@lasers if you're okay with that, merge away thanks!

File: py3status/modules/xkb_layouts.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ class Py3status:
     button_next = 4
     button_prev = 5
     cache_timeout = 10
-    format = "Xkb [\?color=cyan {s}][\?color=darkcyan  {v}]"
+    format = "[\?color=cyan {s}][\?color=darkcyan {v}]"
```
Comment (lasers) [inline py3status/modules/xkb_layouts.py]: This omits a space. Try `setxkbmap -layout us,us -variant ,dvorak`. Also, the tests failed (non-identical formats).

Comment (ultrabug) [reply py3status/modules/xkb_layouts.py]: I fixed it, good catch

I also aligned colors with keyboard_layout

**Comment by lasers** [comment]
@ultrabug Huh. No `Xkb `. :-( Was making it consistent with other modules (i.e. `timer`, `loadavg`, etc). 

I simplified the thresholds. You can merge this because I think you might want a different title.

Nice work, cheers. If this is eerily similar to `keyboard_layout`, should we drop that module too?

**Comment by ultrabug** [comment]
We'll see about removing the old one later

File: py3status/modules/xkb_layouts.py (hunk 1 above)
Comment (lasers) [reply py3status/modules/xkb_layouts.py]: I just realized something. You copying `thresholds` from keyboard_layouts would colorize only the known layouts. The other layouts not in thresholds (as well as the variants.... and other settings) wouldn't be colorized.

The basic `cyan` + `darkcyan` plays it safe and colorizes everything the same regardless.

Just want to share this small but potentially significant information. Cheers.

**Pull request merged**